### ISSUE TITLE
You should be able to access the "Default Colour Settings tab" from the colour picker. 

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#1638]
**The likely journey of the user will be:**
1. User is editing a chart
2. User wants to change colours of the chart so will open the colour picker
3. User is not happy with default settings and wants to change them

Their immediate thought _won't_ be to look in the Settings tab. We should allow them access to this tab from the colour picker itself.

We should add a little settings icon in the colour picker itself to direct them to the right place. ( i have nooo idea if this is even possible!) 
<img width="233" alt="Screenshot 2022-04-01 at 10 04 59" src="https://user-images.githubusercontent.com/12776522/161232130-e2c36fac-0de5-4bd5-ba3e-9464d98d9476.png">

**Mockup**
https://www.figma.com/file/Mhuwki1Bj1akQIwPyr1gWw/Lightdash-product?node-id=2326%3A43462

**Comment by nathaliabj** [comment]
@PriPatel I don't think this is possible we are using [this library](http://casesandberg.github.io/react-color/#create-helpers) which already gives us the block to pick it up and its not very flexible to add anything to it. 

Im thinking there is 2 ways we could do this. 
1. re think a way that the user could access this within the chart config but outside the colour picker block pop up.
2. or we could create our own colour picker component. this will though require more time. 

Let me know what you decide :)

**Comment by rephus** [comment]
This is interesting, how did you get the `cog` icon in the colorpicker ? maybe that button could trigger an event that we can capture and open the panel..

![image](https://user-images.githubusercontent.com/1983672/161493538-e7f71485-168f-4295-a4a0-b674f57fc040.png)

**Comment by rephus** [comment]
This ticket is really old, not sure if it is relevant anymore. @PriPatel  ?

**Comment by PriPatel** [comment]
I think it will be relevant when we work on bigger chart config changes but at the moment it doesnt seem to be a problem for anyone!

**Comment by PriPatel** [comment]
Very relevant if we want to be able to add more flexibility to colours!
